### PR TITLE
reorganize

### LINKED ISSUE #11: Cleanup Repo
DELETE places and airport folders

PULL UP files in mergedFiles to root OR RENAME mergedFiles to src, but place Makefile in root for easier building.

optional:

create sub-folders in src or root for each component (ex: src/airport) to reduce clutter. Remember to change pathnames in Makefile for src code.

### PR DESCRIPTION
**Pull request by galaxyonfire**
Closes #11

Edit: May be easier to view changes by looking at https://github.com/Arjunica/hw4/tree/cleanupRepo

### CODE REVIEW

File: unit_tests/airport_server_unit_tests.c [hunk 1]
```diff
@@ -6,14 +6,14 @@
 #include <assert.h>
```
Comment (QuinnWass) [inline unit_tests/airport_server_unit_tests.c:6]: Lets remove merged files and make a unit tests dir and a data dir

**Review by QuinnWass — CHANGES_REQUESTED** [review]

File: unit_tests/airport_server_unit_tests.c (hunk 1 above)
Comment (galaxyonfire) [reply unit_tests/airport_server_unit_tests.c:6]: merged files and a new unit tests dir are already part of the pr, I can make a data dir for the places/airport info

**Review by Arjunica — COMMENTED** [review]
It looked fine to me

**Review by QuinnWass — APPROVED** [review]

**Comment by QuinnWass** [comment]
Make sure it still runs fine :)

**Comment by galaxyonfire** [comment]
image
@QuinnWass getting this error when trying to run unit tests, any ideas?

**Comment by QuinnWass** [comment]
I had that issue earlier. The vector is empty.

**Comment by QuinnWass** [comment]
Make sure main is merged in right

**Review by QuinnWass — CHANGES_REQUESTED** [review]
Redo this, just deleted places and airport server. Leave merged alone as incremental step.

**Comment by galaxyonfire** [comment]
do you know which commit you did to fix that bug? this branch started after commit 7fbb580f577ed2110d239efae3d85cd98eb24511

**Comment by QuinnWass** [comment]
Main is passing.

**Comment by galaxyonfire** [comment]
@QuinnWass fixed it, do you mind confirming it works on your end?

**Review by Arjunica — CHANGES_REQUESTED** [review]
you will need to merge the latest code from main into this branch before review. ALSO in lieu of 4800 class, lets get rid of testing and just do code reviews 💯

**Comment by QuinnWass** [comment]
Try reverting to  main, pull in changes, rm the places and airport folders, and git mv the files. Git will recognize the move instead of delete

**Comment by galaxyonfire** [comment]
what do you guys think of renaming mergedFiles to src? Additionally, do we want to lift the Makefile to root or just keep it in the folder?

**Comment by QuinnWass** [comment]
Great idea

**Review by QuinnWass — APPROVED** [review]

**Review by Arjunica — APPROVED** [review]
Approved. I saw one issue in the how to run section but I can just fix that in my branch when I do the readme

**Pull request merged**